internal/commands/user_test.go

```diff
@@ -107,3 +109,145 @@ func buildSlackMock() *slack.User {
 		},
 	}
 }
+
+func TestGetStringInt64(test *testing.T) {
+	scenarios := []struct {
+		name         string
+		input        string
+		output       int64
+		expectError  bool
+		errorMessage string
+	}{
+		{
+			name:   "Convert string to int",
+			input:  "0",
+			output: 0,
+		},
+		{
+			name:         "Returns error when try to convert",
+			input:        "1.2",
+			expectError:  true,
+			errorMessage: "strconv.ParseInt: parsing \"1.2\": invalid syntax",
+		},
+	}
+
+	for index, scenario := range scenarios {
+		test.Run(
+			fmt.Sprintf("[%d]-%s", index, scenario.name),
+			func(t *testing.T) {
+				result, err := getStringInt64(scenario.input)
+				if scenario.expectError {
+					if err == nil {
+						t.Fatal("an error was expected, but not occurred")
+					}
+					assert.EqualError(t, err, scenario.errorMessage)
+				} else {
+					if err != nil {
+						t.Fatal("an error occurred, but was not expected")
```

Comment:
You could print also the error:

```suggestion
						t.Fatal(
						  "an error occurred, but was not expected\n",
						  "error: ",
						  err,
						)
```

Reply:
Fixed